interfaces/fiveg_gnb_identity/v0/schema.py

```diff
@@ -0,0 +1,37 @@
+"""This file defines the schemas for the provider and requirer sides of the `fiveg_gnb_identity` relation interface.
+
+It must expose two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {
+            "gnb_name": "gnb001",
+            "tac": "551"
+        }
+    RequirerSchema:
+        unit: <empty>
+        app:  <empty>
+"""
+
+from interface_tester.schema_base import DataBagSchema
+from pydantic import BaseModel, Field
+
```

Comment:
I commented in another PR https://github.com/canonical/charm-relation-interfaces/pull/104#issuecomment-1721199473 Code formatting is not checked for interfaces folder, so the coder formatting problems are not detected. 
Need to add 1 blank line here to pass from PEP8.